llvm/include/llvm/CodeGen/GCEmptyBasicBlocks.h

```diff
@@ -0,0 +1,19 @@
+//===-- GCEmptyBasicBlocks.h ------------------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
```

Comment:
Missing the macro to avoid multiple inclusions.
#ifndef LLVM_CODEGEN_GCEMPTYBASICBLOCKS_H
#define LLVM_CODEGEN_GCEMPTYBASICBLOCKS_H